runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePanel.java

```diff
@@ -100,8 +103,10 @@
 	private static final String SKILL_NAME = "SKILL_NAME";
 	private static final String SKILL = "SKILL";
 
-	private static final HiscoreSkill[] SKILL_PANEL_ORDER = new HiscoreSkill[]
-	{
+	/**
+	 * Real skills, ordered in the way they should be displayed in the panel.
+	 */
+	private static final Set<HiscoreSkill> SKILLS = new LinkedHashSet<>(Arrays.asList(
```

Comment:
This seems like this would break the ordering of the panel? As it relies on the order of this. Also `ImmutableSet.of`